source/_posts/azure-purview.md

```diff
@@ -0,0 +1,149 @@
+
+Start out by creating a collection by clicking on "**+ new collection**". The collection requires you to fill in a name. In this example a collection called "westeuropedatasources" was created in which one could store all data sources available in the West Europe region. 
+
+<img src="/images/azure-purview/collection.png" />
+
+After a collection is created, you can start registering data sources. Currently, because of preview the number of supported data sources is limited. However, this list is sure to grow quickly and will also include multi –cloud data sources (i.e. giving you the ability to scan data in AWS) 
+
+<img src="/images/azure-purview/datasources.png" />
+
+For now, we will use the Azure SQL Database to scan. When selecting the Azure SQL Database to scan, it will give the the option to select a database, which will then be registered under sources. 
+
+<img src="/images/azure-purview/collection-region.png" />
+
+### Scanning datasource
+
+To start scanning, press the scan button on the registered datasource.
+
+<img src="/images/azure-purview/start-scan.png" />
+
+The data that has to be filled in is pretty straightforward. Under the credentials, we can select how we want to connect to it. When setting it up for the Purview MSI, it requires extensive permissions to a point where you have to give it database owner permission on the database. This gives it full access to the database.
+
+If this is something that you dont want, you can create a new managed identity.
+
+To create a new managed identity, it requires a name, what kind of authentication method it should use (we selected SQL authentication). No matter what method you use, it will require you to use Azure Keyvault credentials which make sure you don’t hardcode the password anywhere. This is a very nice security feature they included. 
+
+<img src="/images/azure-purview/create-credential.png" />
+
+After having set up the managed identity and given the managed identity access to the keyvault, it is time to start the scan. Select which tables you want to scan and in which interval you want the scan to be executed. You can also create your own custom rulesets, that is outside the scope of this article. 
+
+<img src="/images/azure-purview/sqldatabase.png" />
+
+After the scan is complete, the details on the database pane will show how many scans have run and how many assets it has scanned. 
+
+## Data catalog
+
+After the data source registrations and scans, we can use the results of the scanned data to know more about the data in our organisation. 
+
+We can start on the homepage, which is accessible to the organisation (access can be given to accounts, based on role). When we start searching you can see that we start to see data about the database we just scanned. When typing in customer, it shows the result of the asset it found, which in this case is a table about customer addresses. 
```

Comment:
We can start on the homepage, which is accessible to the organisation (access can be given to accounts, based on role) >> Let's start at the homepage which is accessible to the organisation based on role and account access policies. 
When we start searching you can see that we start to see data about the database we just scanned. >> As we start searching, the data catalog will return data about that database we just scanned  (+metadata instead of data maybe?)
When typing in customer >> When we enter customer (add code markup?)